browser/ui/views/brave_default_browser_dialog_view.cc

```diff
@@ -0,0 +1,156 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/ui/views/brave_default_browser_dialog_view.h"
+
+#include <utility>
+
+#include "base/bind.h"
+#include "base/logging.h"
```

Comment:
is it needed? pls double-check for unused headers

Reply:
Fixed.